### PR TITLE
Add-non-root-test

### PR DESCRIPTION
**Pull request by brunoborges**

### CODE REVIEW

File: docker/test-only/NonRootUser.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class NonRootUser {
```
Comment (karianna) [inline docker/test-only/NonRootUser.java:1]: I would like my Javadoc and some sane whitespace please :-)

File: docker/test-only/NonRootUser.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+ public static void main(String[] args) {
+  String expectedUser = "app";
+  String user = System.getProperty("user.name");
+  assert user != null : "user.name is null";
```
Comment (karianna) [inline docker/test-only/NonRootUser.java:5]: I get it's likely not worth adding a Testing lib for this, but perhaps consider for future tests?  Logging also.

**Review by karianna — CHANGES_REQUESTED** [review]
GH Actions failing on validate phase.

File: docker/test-only/NonRootUser.java (hunk 2 above)
Comment (brunoborges) [reply docker/test-only/NonRootUser.java:5]: Not sure. The goal is not to test the Java application, but the container image.

File: docker/test-only/NonRootUser.java (hunk 1 above)
Comment (brunoborges) [reply docker/test-only/NonRootUser.java:1]: Yeah yeah... VS Code is not formatting adequately at the moment. :/

Comment (brunoborges) [reply docker/test-only/NonRootUser.java:1]: Formatting is fixed. :)

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**